chromium_src/url/base58.h

```diff
@@ -0,0 +1,119 @@
+// Copyright (c) 2014-2019 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
```

Comment:
cc @fmarier on if we need to do anything here to comply with the license

Reply:
for maintainability and auditability, we strongly prefer that third party code be included in a way such that it's easy to see the diff that is added by Brave: https://github.com/brave/internal/issues/400.

see https://bravesoftware.slack.com/archives/G2KN13Z8C/p1532742342000007 for more context

Reply:
changed it to use existing implementation in brave/third_party/bitcoin-core/src/src/base58.h

Reply:
For posterity, if we add a new piece of third-party code, then a `README.chromium` file needs to be added, typically in `src/brave/third_party`. For example: https://github.com/brave/brave-core/blob/master/third_party/bitcoin-core/README.chromium